src/components/PostTable.js

```diff
@@ -1,13 +1,125 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const TableWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 12px;
+  padding-bottom: 127px;
+`;
+
+const PostsTable = styled.table`
+  max-width: 786px;
+  width: 100%;
+  margin: -2 auto 0;
+  cell-collapse: 0;
+  border-spacing: 0;
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+`;
+
+const TableHeading = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  max-width: 786px;
+  width: 100%;
+  margin: 0 auto;
+`;
+
+const Th = styled.th`
+  text-align: left;
+  font-size: ${({ theme }) => theme.font.size.small};
+  color: ${({ theme }) => theme.color.secondary};
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+  padding: 8px 12px;
+  white-space: nowrap;
+`;
+
+const Td = styled.td`
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+  padding: 8px 12px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: ${({ theme }) => theme.color.secondary};
+  font-size: ${({ theme }) => theme.font.size.small};
+`;
+
+const TitleTd = styled(Td)`
+  max-width: 350px;
```

Comment:
Great that you set the max-width here and for the author column :+1: 

Setting the `width` as well is required to keep the title column the desired size in case all titles are very short.

Reply:
I've ended up doing something a little differently after I had trouble getting things to be the exact width I wanted them to be.  Let me know if that's okay!